test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -739,6 +739,9 @@ dir -Recurse `
                 @{ inputStr = "Get-Process >'.\My ``[Path``]\'"; expected = "'.${separator}My ``[Path``]${separator}test.ps1'" }
                 @{ inputStr = "Get-Process >${tempDir}\My"; expected = "'${tempDir}${separator}My ``[Path``]'" }
                 @{ inputStr = "Get-Process > '${tempDir}\My ``[Path``]\'"; expected = "'${tempDir}${separator}My ``[Path``]${separator}test.ps1'" }
+                @{ inputStr = "cd 'My ``["; expected = "'.${separator}My ``[Path``]'" }
```

Comment:
1. Updated test.
2. I disagree. User should type escape for `-Path` (which support globbing), just like we do in bash (eg. `touch \[test\]`). But if you typed escape and auto completion still did not work, that is because by default, powershell treats "unquoted string without space" like "double-quoted string", that "double-quoted string" will consume escape character. For file name `[test]`, either ````Get-Content -Path ```[```` or ``Get-Content -Path '`[`` will work.
3. Would you please let me know what did not work for `-LiteraPath`?
I tried `Get-Item -LiteralPath [`(TAB) for a file named `[test]` and it did auto complete.

Reply:
2. Consider interactive UX. User do `dir` and then type directory or file name as one see on screen - without escapes. In the scenario both the user and PowerShell know right name. And more, we can have both names - with escape char too.
3. Get-Item '.\My `[

Reply:
2. The problem is, `-Path` support wildcard. When a path can be an expression, the special characters should always be escaped otherwise it will become too complicated. For example, in a directory there are `A.txt`, `B.txt` and `[AB].txt`, what should `Get-Item -Path [AB].txt` return?
3. I tested both `-Path` and `-LiteralPath` and both will complete the file name. eg.
`New-Item -Name 'My [test]'`
``Get-Item '.\My `[`` will be completed to ``Get-Item '.\My `[test`]'``, and
``Get-Item -LiteralPath '.\My [`` will be completed to ``Get-Item -LiteralPath '.\My [test]'``
Did you use the build here? https://ci.appveyor.com/project/PowerShell/powershell/build/v6.1.0-rc.10960/job/3i76g2ar1wqlvw7x/artifacts